### PR TITLE
feat: add text

### PR DESCRIPTION
**Pull request by Tatyana97**

### CODE REVIEW

**Comment by buzlan** [comment]
1) Libraries that we import from the react itself, put at the top, then indent then import the rest (rules of good code)
first screenshot not good! The second is good!
Screen Shot 2022-04-14 at 11 23 08 PM
Screen Shot 2022-04-14 at 11 28 20 PM

2) Dont use undefined like default value of the state. 
The best practice would be to use null.

Create folder services, where will you store information about api adress. In this folder, create index.js  file, where you will do export const API = "your path" and use import at the main page for got this variable.
Screen Shot 2022-04-14 at 11 30 57 PM

3) remove unused code, e.g. 65 - 72 lines, and remove all logs.


!IMPORTANT
fix my remarks and let me know when you upload the changes to the branch, pr will be updated automatically.
Good luck))

File: src/Componets/MainPage/MainPage.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ class MainPage extends React.Component {
 			 city: undefined
 			})
 			console.log(err);
+			console.log("fdfd");
 		}
-	
 	  }
```
Comment (buzlan) [inline src/Componets/MainPage/MainPage.js:74]: fix remarks from comments

File: src/Componets/MainPage/MainPage.js [hunk 2]
```diff
@@ -51,37 +49,25 @@ class MainPage extends React.Component {
...
-			// 	  temp: undefined,
-			// 	  city: undefined,
-			// 	  country: undefined,
-			// 	  time:undefined,
-			// 	  error: "Введите название города"
-			// 	});
 			  }	
 		} catch (err) {
 			this.setState({
 			 error: "Не найден город",
```
Comment (buzlan) [inline src/Componets/MainPage/MainPage.js]: add text "Не найден город" to constants. and use variable here like :
error: errorText.

File: src/Componets/MainPage/shared/error.js [hunk 3]
```diff
@@ -4,6 +4,5 @@ import React from 'react';
 export const Error = props =>(
```
Comment (buzlan) [inline src/Componets/MainPage/shared/error.js:6]: add styles to this component, e.g. Red text, or will good do it like push notification ,e.g. 
you can use this library for https://www.npmjs.com/package/react-notifications-component

File: src/Componets/MainPage/weather.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```
Comment (buzlan) [inline src/Componets/MainPage/weather.js:2]: Why you have <p> Error, if u not use it? line 13

**Review by buzlan — COMMENTED** [review]
fixed remarks and ping me

File: src/Componets/MainPage/shared/error.js (hunk 3 above)
Comment (Tatyana97) [reply src/Componets/MainPage/shared/error.js:6]: I use simple css or better use library react-notifications-component?

Comment (buzlan) [reply src/Componets/MainPage/shared/error.js:6]: better use library react-notifications-component, or u can use another library if u want:)

**Review by buzlan — APPROVED** [review]

**Pull request merged**